tests/x11regressions/gnomecase/application_starts_on_login.pm

```diff
@@ -131,22 +128,29 @@ sub run {
     ##auto-save-session functionality has been abandoned;
     ##current status: just firefox works
     ##so in the future will consider remove openqa code for this session
+    if (sle_version_at_least('15') || check_var('VERSION', 'Tumbleweed')) {
```

Comment:
better to exclude the older versions explicitly, e.g. `if ((is_sle && !sle_version_at_least('15')) && !leap_version_at_least('15'))`

Reply:
Sorry, my mistake. I want to install dconf-editor for TW (the app is installed by default in SLED), check_var('VERSION', 'Tumbleweed') is enough. I will fix it.